platform/product-management/release-plan-template.md

```diff
@@ -6,6 +6,38 @@ So: **how might we craft a release plan to test our product "in the wild" at a s
 
 That's what this Release Plan Template is for! And note - there are feature toggles and beta banners at your disposal that you can use as a part of your plan.
 
+## Do I need unmoderated production testing?
+
+**Yes**, staged rollout is required unless you can confidently answer "yes" to all of the following:
+
+* This change does not add substantial new functionality to VA.gov
+* This change does not impact user flows through tasks
+* This change does not affect traffic to backend services
+
+*Example*: a change to a page's text content **could skip** staged rollout
+
+*Example*: a visual redesign to a page that doesn't affect user flows **could skip** staged rollout
+
+*Example*: adding a new field to an existing form, that doesn't **could skip** staged rollout
+
+*Example*: a new feature on an existing application that creates new backend traffic **needs staged rollout**
+
+*Example*: a significant change to how users navigate an existing form **needs staged rollout**
+
+*Example*: a feature that will route significantly more users (and therefore more backend traffic) to an existing application **neds staged rollout**
+
+### Exceptions
+
+If [feature toggles](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/platform/tools/feature-toggles/) don't work for your use case, you can request an exception in Staging Review.
+
+| Feature type | Possible with feature toggles? |
+| --- | --- |
+| New feature in existing application | Yes |
+| New application | Yes |
+| Static content changes | Doable but tricky |
+| URL redirects | No |
+
+Other exceptions to this requirement can be approved by DEPO VSP leads.
```

Comment:
This line is showing up as a header in preview due to formatting. Doesn't need to be a header.